realsense2_camera/src/actions.cpp

```diff
@@ -0,0 +1,121 @@
+// Copyright 2024 Intel Corporation. All Rights Reserved.
```

Comment:
actions.cpp is a generic name.
Do we want a calibration action dedicated name instead ?

Reply:
I think we will add all actions in the future inside it (we can add FW-update action for example, etc..)
(In the same way, I will create a "services.cpp" file for all services - we can redesign this in a separate task)

Reply:
On next action/service please consider creating a service/action folder and implement a file for each serve/action
With time it may grow..